tests/node_account/test_transfer_tokens_from_frozen_to_unfrozen.py

```diff
@@ -0,0 +1,33 @@
+"""
+Provide tests for command line interface's node account transfer tokens from frozen to unfrozen operational balances.
```

Comment:
`transfer available tokens`

Reply:
@Alladin9393, if I add it, the line's length will be more that 120.